src/tox/config/__init__.py

```diff
@@ -987,18 +973,23 @@ def __init__(self, config, ini_path, ini_data):  # noqa
         config.hashseed = hash_seed
 
         reader.addsubstitutions(toxinidir=config.toxinidir, homedir=config.homedir)
+
         # As older versions of tox may have bugs or incompatibilities that
         # prevent parsing of tox.ini this must be the first thing checked.
         config.minversion = reader.getstring("minversion", None)
         if config.minversion:
-            tox_version = pkg_resources.parse_version(tox.__version__)
-            config_min_version = pkg_resources.parse_version(self.config.minversion)
-            if config_min_version > tox_version:
-                raise tox.exception.MinVersionError(
-                    "tox version is {}, required is at least {}".format(
-                        tox.__version__, self.config.minversion
+            # As older versions of tox may have bugs or incompatibilities that
+            # prevent parsing of tox.ini this must be the first thing checked.
+            config.minversion = reader.getstring("minversion", None)
+            if config.minversion:
+                tox_version = pkg_resources.parse_version(tox.__version__)
+                config_min_version = pkg_resources.parse_version(self.config.minversion)
```

Comment:
I'd love for us to get away from `pkg_resources` at some point, but probably not today 😆

Reply:
indeed 👍 any better suggestions to detect what's installed and what's not?